recipes/date/all/conanfile.py

```diff
@@ -87,6 +89,7 @@ def package(self):
             dst = os.path.join("include", "date")
             self.copy(pattern="date.h", dst=dst, src=src)
             self.copy(pattern="tz.h", dst=dst, src=src)
+            self.copy(pattern="ptz.h", dst=dst, src=src)
             self.copy(pattern="iso_week.h", dst=dst, src=src)
             self.copy(pattern="julian.h", dst=dst, src=src)
             self.copy(pattern="islamic.h", dst=dst, src=src)
```

Comment:
I feel like this is easier 🌮 

```suggestion
            for file in ["date.h", "tz.h", "ptz.h", "iso_week.h", "julian.h", "islamic.h"]:
                self.copy(pattern=file, dst=dst, src=src)
```

Reply:
Hi. This is required? Because will reset reviews, build etc.

Reply:
I can add it after this PR, because this PR is required to use this library correctly. Can we do in this way?

Reply:
No required, I approved! hopefully next time this recipe gets updated ... there's a few out of date conventions

Reply:
at university, on 1st semester, there was an exercise about for loop, the idea was print from 1 to 10 using C language. A colleague preferred to write printf 10 times, each line for a single number ... don't try it at home.